src/bin/ch-remote.rs

```diff
@@ -661,7 +661,7 @@ fn dbus_api_do_command(matches: &ArgMatches, proxy: &DBusApi1ProxyBlocking<'_>)
             proxy.api_vm_snapshot(&snapshot_config)
         }
         Some("restore") => {
-            let restore_config = restore_config(
+            let (restore_config, _fds) = restore_config(
```

Comment:
Hey @likebreath, thank you for bringing this to my attention. For the D-Bus API, `ch-remote` indeed completely ignores fds passed to `add-net`, if any. It seems that providing an alternative to `simple_api_command_with_fds` is actually a TODO that I missed implementing.

I'll look into fixing this behavior to function as expected as soon as possible. Do you have any recommendations for addressing this issue in the previous versions of CH, since the behavior has been present since the first introduction of the D-Bus API?